src/Analysis/Engine/Impl/AnalysisValueSetExtensions.cs

```diff
@@ -140,8 +135,7 @@ public static class AnalysisValueSetExtensions {
         /// Performs a delete index operation propagating the index types into
         /// the provided object.
         /// </summary>
-        public static void DeleteIndex(this IAnalysisSet self, Node node, AnalysisUnit unit, IAnalysisSet index) {
-        }
+        public static void DeleteIndex(this IAnalysisSet self, Node node, AnalysisUnit unit, IAnalysisSet index) { }
```

Comment:
Empty static method?

Reply:
Apparently it is called from DDG... Let's leave this alone until we decide fate of the DDG.